### PR TITLE
Some updates to the README.

### PR DESCRIPTION
**Pull request by glennc**
Making them closer to the dotnet image and adding an example to the build image.

@natemcmaster @richlander

### CODE REVIEW

File: 1.1.0/jessie/build-projectjson/README.md [hunk 1]
```diff
@@ -8,24 +8,43 @@ projects. These Dockerfiles use the [microsoft/dotnet](https://hub.docker.com/r/
...
 - [`1.0.1`, (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build-projectjson/Dockerfile)
 
+## What is ASP.NET Core?
+
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud based internet connected applications, such as web apps, IoT apps and mobile backends. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at [GitHub](https://github.com/aspnet). 
+
+This image contains:
+
+- [.NET Core SDK](https://github.com/dotnet/cli) so that you can create, build and run your .NET Core applications.
+- A Nuget package cache for the ASP.NET Core libraries.  This will significantly improve the initial package restore performance when building ASP.NET Core application.
```
Comment (natemcmaster) [inline 1.1.0/jessie/build-projectjson/README.md]: Nit: capital G. 'NuGet'

File: 1.1.0/jessie/build-projectjson/README.md [hunk 2]
```diff
@@ -8,24 +8,43 @@ projects. These Dockerfiles use the [microsoft/dotnet](https://hub.docker.com/r/
...
+- [Bower](https://bower.io/)
+- [Gulp](http://gulpjs.com/)
+
 ## Example Usage
 
+### Build on run
+
+You can use this container to compile your application when it runs. If you use the [Visual Studio tooling](https://blogs.msdn.microsoft.com/webdev/2016/11/16/new-docker-tools-for-visual-studio/) to setup CI/CD to Azure Container Service then this method of using the build container is used.
+
+1. Run the build container, mounting your code and output directory, and publish your app:
```
Comment (natemcmaster) [inline 1.1.0/jessie/build-projectjson/README.md]: Seems odd to have a numbered list of one item. Maybe leave as a normal paragraph?

File: 1.1.0/jessie/build-projectjson/README.md [hunk 3]
```diff
@@ -8,24 +8,43 @@ projects. These Dockerfiles use the [microsoft/dotnet](https://hub.docker.com/r/
...
+
+```
+docker run --it -v $(PWD):/app --workdir /app microsoft/aspnetcore-build bash -c "dotnet restore && dotnet publish -c Release -o ./bin/Release/PublishOutput"
+```
+
+After this has run the application in the current directory will be published to the `bin/Release/PublishOutput` directory.
+
+### Build on build
```
Comment (natemcmaster) [inline 1.1.0/jessie/build-projectjson/README.md]: Maybe "Build on `docker-build`"? Too many builds

File: 1.1.0/jessie/build-projectjson/README.md [hunk 4]
```diff
@@ -8,24 +8,43 @@ projects. These Dockerfiles use the [microsoft/dotnet](https://hub.docker.com/r/
...
+
+- [.NET Core SDK](https://github.com/dotnet/cli) so that you can create, build and run your .NET Core applications.
+- A Nuget package cache for the ASP.NET Core libraries.  This will significantly improve the initial package restore performance when building ASP.NET Core application.
+- [Node.js](https://nodejs.org)
+- [Bower](https://bower.io/)
+- [Gulp](http://gulpjs.com/)
+
 ## Example Usage
 
+### Build on run
```
Comment (natemcmaster) [inline 1.1.0/jessie/build-projectjson/README.md]: Suggesting 'Build on "docker-run"' to distinguish from dotnet-run.

**Comment by natemcmaster** [comment]
Resolves https://github.com/aspnet/aspnet-docker/issues/167

**Comment by glennc** [comment]
Responded to @natemcmaster feedback. Added a related images section to address issues like this: https://github.com/dotnet/dotnet-docker/issues/155#issuecomment-260314044 until we have something better.

File: 1.1.0/jessie/build-projectjson/README.md [hunk 5]
```diff
@@ -8,24 +8,48 @@ projects. These Dockerfiles use the [microsoft/dotnet](https://hub.docker.com/r/
...
-- [`1.1.0-projectjson`, `latest` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-projectjson/Dockerfile)
-- [`1.1.0-msbuild` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1.0/jessie/build-msbuild/Dockerfile)
-- [`1.0.1-msbuild`, (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build-msbuild/Dockerfile)
-- [`1.0.1`, (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0.1/jessie/build-projectjson/Dockerfile)
+1. [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/)
+2. [microsoft/aspnetcore](https://hub.docker.com/r/microsoft/aspnetcore/)
 
 ## Example Usage
 
+### Build an app on `docker run`
```
Comment (richlander) [inline 1.1.0/jessie/build-projectjson/README.md]: > Build an app on `docker run`

The use of "on" feels on. How about "with"? Same thing for the section below.

File: 1.1.0/jessie/build-projectjson/README.md [hunk 6]
```diff
@@ -52,6 +76,6 @@ This image contains:
     $ docker cp build-cont:/out ./output
     ```
 
-Now you have built your ASP.NET Core application inside a container and have the published output on the host ready to
-deploy. From here you could then construct an optimized runtime image with the `microsoft/aspnetcore` image or just
-deploy/run the binaries as normal without using Docker at runtime.
\ No newline at end of file
+Now you have built your ASP.NET Core application inside a container and have the published output on the host ready to deploy. From here you could then construct an optimized runtime image with the `microsoft/aspnetcore` image or just deploy/run the binaries as normal without using Docker at runtime.
```
Comment (richlander) [inline 1.1.0/jessie/build-projectjson/README.md]: The "run" example results in a running app. It seems like the "build" example should, too.

**Comment by richlander** [comment]
I just read the Title and intro paragraph for both image types. I think we need to do more differentiation on both to make it crystal clear how the two repos differ.

File: 1.1.0/jessie/build-projectjson/README.md (hunk 6 above)
Comment (glennc) [inline 1.1.0/jessie/build-projectjson/README.md]: This does that as well, in the output directory is a fully published app that you can run with `dotnet run`. I will try to make that clearer.

**Pull request merged**